test/TestUtilities/Test.Utility/Test.Utility.csproj

```diff
@@ -3,42 +3,27 @@
     <TargetFrameworks>$(TargetFrameworksUnitTestForSigning)</TargetFrameworks>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <NoWarn Condition=" $(TargetFramework.StartsWith('netcoreapp')) ">$(NoWarn);CS1998</NoWarn>
-    <PackProject>true</PackProject>
-    <Shipping>true</Shipping>
-    <IsPackable>true</IsPackable>
+    <PackProject>false</PackProject>
+    <Shipping>false</Shipping>
+    <IsPackable>false</IsPackable>
```

Comment:
NuGet server team uses this Test.Utility package for signed package testing of nuget.org ingestion pipelines. This needs to keep being a package for them.

Reply:
It looks like its only used in NuGet.Jobs and version 6.4?  Would it be better to get them a better system or continue to "ship" this test utility?  They obviously don't need an update, unless there are more usages I couldn't find.

https://github.com/search?q=org%3ANuGet%20%22Test.Utility%22&type=code

Reply:
They're using the package creating utilities. 
I think we should keep it for now and not create potentially extra work for them. 
IMO we can remove if they ok it.

Reply:
Okay I'll bring it back